scripts/install_mhnserver_unattended.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+if test -f "../server/config.py"; then
+    ./install_mhnserver.sh
```

Comment:
I think this will break if someone tries to run the script from another directory.  I recommend adding this to the top:

```
scripts_directory=$(dirname $(readlink -f $BASH_SOURCE))

# ... and later

$scripts_directory/install_mhnserver.sh
```